redux/reducers.js

```diff
@@ -0,0 +1,60 @@
+import { combineReducers } from 'redux';
+import * as types from "./actionTypes";
+
+/* Current shape of state:
+ {
+  user: "David",
+  currentChat: id 
+  chats: [ { id: 1,
+    name: 'Tea Chats',
+    users: ['David', 'Nick'] },
+    { id: 2,
+      name: 'Coffee Chats',
+      users: ['David', 'Ryan'] } ]
+  messages: [
+    { id: 0, chats_id: 1, user: 'David', message: 'ayyyyy', emojis: { David: ':joy:', Nick: ':laughing:' } },
+    { id: 1, chats_id: 1, user: 'Nick', message: 'lmao', emojis: {} }
+  ]
+} */
+
+const messageInitialState = {
+  messages: []
+};
+
+const userInitialState = {
+  user: '',
+  currentChat: 0,
+  chats: []
+};
+
+function messageReducer(state = messageInitialState, action) {  
+  console.log("messageReducer");
+  console.log(state); 
+    switch (action.type) {
+    case types.ADD_MESSAGE:
```

Comment:
`case` statements should be indented in since they are part of/enclosed in the `switch`.

Reply:
My bad, should I be using 4 spaces for just the indentation like this? I think I can just double-tab in VScode.

Reply:
You shouldn't have to manually use spaces, but yeah, your tabs should be set to 4 spaces apiece.